bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v2/PageImpl.java

```diff
@@ -116,6 +125,15 @@ private String getProxyPath(ClientLibrary lib) {
         return path;
     }
 
+    protected void populateClientLibCategoriesJs() {
+        if (currentStyle != null) {
+            clientLibCategoriesJsHead = currentStyle.get(PN_CLIENTLIBS_JS_HEAD, ArrayUtils.EMPTY_STRING_ARRAY);
+            LinkedHashSet<String> categories = new LinkedHashSet<>(Arrays.asList(clientLibCategories));
+            categories.removeAll(Arrays.asList(clientLibCategoriesJsHead));
+            clientLibCategoriesJsBody = categories.toArray(new String[0]);
```

Comment:
`clientLibCategoriesJsBody` might contain CSS clientlibs, right? So I would call it `clientLibCategoriesBody`.

Reply:
@jckautzmann - it could contain CSS clientlib categories, but they wouldn't be loaded. This function is supposed to return JS clientlibs for the page body. We need the `Js`